test/unit_inbroker_parallel_SUITE.erl

```diff
@@ -1018,6 +1041,161 @@ set_vm_memory_high_watermark_command1(_Config) ->
                     )
     end.
 
+max_length_drop_head(Config) ->
+    {_Conn, Ch} = rabbit_ct_client_helpers:open_connection_and_channel(Config, 0),
+    QName = <<"max_length_drop_head_queue">>,
+    QNameDefault = <<"max_length_default_drop_head_queue">>,
+    QNameBytes = <<"max_length_bytes_drop_head_queue">>,
+    QNameDefaultBytes = <<"max_length_bytes_default_drop_head_queue">>,
+
+    MaxLengthArgs = [{<<"x-max-length">>, long, 1}],
+    MaxLengthBytesArgs = [{<<"x-max-length-bytes">>, long, 100}],
+    OverflowArgs = [{<<"x-overflow">>, longstr, <<"drop-head">>}],
+    amqp_channel:call(Ch, #'queue.delete'{queue = QName}),
```

Comment:
If cleanup is properly implemented, these `queue.delete` shouldn't be needed at the beginning of the test. Or am I missing something? I think instead `end_per_group` these changes require `end_per_testcase`.